tests/util/translator/translator.spec.tsx

```diff
@@ -0,0 +1,311 @@
+import { EElementType, TElementMap } from "src/context-providers";
+import { ELEMENT_VALIDATION_TYPES } from "src/data";
+import { Translator } from "src/translator";
+import {
+    generateMockElement,
+    generateMockElementSchema,
+    generateMockSchema,
+} from "./helper";
+
+describe("Translator", () => {
+    beforeEach(() => {
+        global.ResizeObserver = jest.fn().mockImplementation(() => ({
+            observe: jest.fn(),
+            unobserve: jest.fn(),
+            disconnect: jest.fn(),
+        }));
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+    describe("generateSchema", () => {
+        describe('Email field schema generation"', () => {
```

Comment:
- use double quotes for test description
- errant double quote at the end of the description
- do this for all the test descriptions in this file
```suggestion
        describe("Email field schema generation", () => {
```